packages/fxa-payments-server/src/routes/Subscriptions/PaymentUpdateForm.test.tsx

```diff
@@ -94,6 +97,55 @@ describe('routes/Subscriptions/PaymentUpdateFormV2', () => {
     expect(screen.queryByTestId('paymentForm')).toBeInTheDocument();
   });
 
+  it('renders correctly for paypal', async () => {
+    render(<Subject customer={{ ...CUSTOMER, payment_provider: 'paypal' }} />);
+    expect(
+      screen.getByTestId('change-payment-update-button')?.getAttribute('href')
+    ).toEqual('https://www.paypal.com/myaccount/autopay/connect/ba-131243');
+  });
+
+  it('renders correctly for missing billing agreement for paypal', async () => {
+    render(
+      <Subject
+        customer={{
+          ...CUSTOMER,
+          payment_provider: 'paypal',
+          paypal_payment_error: 'missing_agreement',
+        }}
+      />
+    );
+    expect(
+      screen.queryAllByTestId('reveal-payment-modal-button').length
+    ).toEqual(2);
```

Comment:
I don't understand why `2` here.

Reply:
Same as above, it is rendered twice: once in the notification and once in the subscription view

Reply:
Ah.  Sorry, I think the older comment was marked as resolved and it auto-collapsed by Github.